cognite/client/_api/functions.py

```diff
@@ -466,8 +465,6 @@ def _zip_and_upload_folder(self, folder: str, name: str, external_id: Optional[s
             file_id = cast(int, file.id)
 
             return file_id
-        except Exception as e:
```

Comment:
Any reason why this is removed?

Reply:
Looks redundant to me, unless there is some behaviour this ensures which I am missing?